src/components/Card/tests/Card.test.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Card from '../Card';
+
+describe('<Card/>', () => {
+  it('should render a card with color and text', () => {
+    const size = { width: 100, height: 200 };
+    const { getByText } = render(<Card text="Hello World" color="yellow" size={size} />);
+    expect(getByText('Hello World')).toMatchSnapshot();
+  });
+});
```

Comment:
so far, testing library seems easy enough :D

Why are we only snapshotting the div that contains the text? Should we not snapshot the whole thing?

Reply:
yeah you're right, I should be getting the snapshot of the whole container